.vscode/settings.json

```diff
@@ -1,8 +1,10 @@
 {
```

Comment:
why the changes here? did you run the project with ./gradlew run? or vscode run? readme.txt explains it for you if you have any questions